sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/ChangeFeedObserver.java

```diff
@@ -34,4 +35,6 @@ public interface ChangeFeedObserver {
      * @return a deferred operation of this call.
      */
     Mono<Void> processChanges(ChangeFeedObserverContext context, List<JsonNode> docs);
+
+    Mono<Void> processChangesV1(ChangeFeedObserverContext context, List<ChangeFeedProcessorItem> docs);
```

Comment:
Nit: add a TODO here to come back and refactor the code. "V1" in the name is not my first choice given that we only return the new ChangeFeedProcessorItem for FF.